feat: create node exe from rclcpp components

wep21 opened the pull request:
Signed-off-by: wep21 <border_goldenmarket@yahoo.co.jp>

clydemcqueen commented:
Thank you for the contribution @wep21 !

It appears that I was exporting the node twice under 2 names (gscam and gscam_node). This change does indeed simplify matters.

However, several example launch files (e.g., `v4l.launch`) are still look for the node under the name `gscam`. Would you be able to fix those as well? If not, I will update the launch files.

Thanks again.

wep21 commented:
@clydemcqueen Thank you for your suggestion. I fixed executable name in launch file and added some minor modification.
I appreciated it if you could check these changes.

clydemcqueen commented:
Excellent improvements, thank you. Merging.
